src/components/tasks/card/index.tsx

```diff
@@ -473,12 +473,12 @@ const Card: FC<CardProps> = ({
                             Estimated completion
                         </span>
                         <span className={classNames.completionDate}>
-                            {renderDate(fromNowEndsOn, shouldEdit)}
+                            {renderDate(fromNowEndsOn, isEditable)}
```

Comment:
- as we're returning JSX here will it be better to use this as a react component and not a function call ?

Reply:
agreed with you on this but is a separate refactor task, doesn't come under the scope of this PR

Reply:
- viewing the file I am not able to see the `renderDate` being used it in once place that is here, can we do this in this pr as it does not add much to the scope?

Reply:
No yash it's not just rename if I rename this there are other minor refactors also, and I don't want to address them in this PR, refactor is being addressed separately

Reply:
- got it, in that case can you please link the refactor ticket in the pr.

Reply:
https://github.com/Real-Dev-Squad/website-status/issues/497